### PR TITLE
geoloc and searchbar fusioned

### PR DESCRIPTION
**Pull request by guigzzz31**

### CODE REVIEW

File: src/components/MapTest.jsx [hunk 1]
```diff
@@ -1,33 +1,63 @@
...
+        haveUsersLocation: true,
+        zoom: 13
+      });
+    });
+  }
+
   render() {
+    const position = [this.state.location.lat, this.state.location.lng];
+    const isUserLocation = this.state.haveUsersLocation;
+    const voilaZoom = isUserLocation ? this.state.zoom : this.props.zoomCity;
```
Comment (akopsykokwak) [inline src/components/MapTest.jsx]: Need better name for variable

File: src/components/MapTest.jsx [hunk 2]
```diff
@@ -1,33 +1,63 @@
...
+    const voilaCenter = isUserLocation ? position : this.props.cityCenter;
```
Comment (akopsykokwak) [inline src/components/MapTest.jsx]: Need better name for variable

**Review by akopsykokwak — CHANGES_REQUESTED** [review]

File: src/components/MapTest.jsx (hunk 1 above)
Comment (guigzzz31) [reply src/components/MapTest.jsx]: ok

File: src/components/MapTest.jsx (hunk 2 above)
Comment (guigzzz31) [reply src/components/MapTest.jsx]: ok

**Review by akopsykokwak — APPROVED** [review]

**Review by Froyj — APPROVED** [review]

**Pull request merged**